doc/source/whatsnew/v1.2.0.rst

```diff
@@ -677,6 +677,8 @@ Indexing
 - Bug in :meth:`DataFrame.loc` and :meth:`DataFrame.__getitem__`  raising ``KeyError`` when columns were :class:`MultiIndex` with only one level (:issue:`29749`)
 - Bug in :meth:`Series.__getitem__` and :meth:`DataFrame.__getitem__` raising blank ``KeyError`` without missing keys for :class:`IntervalIndex` (:issue:`27365`)
 - Bug in setting a new label on a :class:`DataFrame` or :class:`Series` with a :class:`CategoricalIndex` incorrectly raising ``TypeError`` when the new label is not among the index's categories (:issue:`38098`)
+- Bug in :meth:`Series.loc` and :meth:`Series.iloc` raising ``ValueError`` when inserting a listlike ``np.array``, ``list`` or ``tuple`` in an ``object`` Series of equal length (:issue:`37748`)
```

Comment:
you list 3 issues in the top of PR, what's missing?

Reply:
Ah, I added a whastnew note for #38271 and one for #37748.
In the PR, I listed #37486 because it is a particular case of #37748 (iloc on a Series with a list, both of size 1).

Reply:
Pls add the issue number to the note for #37748 then

Reply:
ok